ci-operator/config/open-cluster-management/insights-client/open-cluster-management-insights-client-release-2.4.yaml

```diff
@@ -0,0 +1,92 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux-amd64
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: insights-client
+promotion:
+  disabled: true
+  name: "2.4"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: publish
```

Comment:
@mqhackett We need to disable publish from travis before we can merge this.

Reply:
@jlpadilla Here is a PR to disbable for 

must-gather: https://github.com/open-cluster-management/must-gather/pull/70

insights-client: https://github.com/open-cluster-management/insights-client/pull/55

Reply:
Thanks, both PRs are merged.  We can merge this one now.